src/components/dashboard/GapByRace.vue

```diff
@@ -86,16 +142,33 @@ const minVaxRace = computed(() => {
 });
 </script>
 
-<style scoped>
+<style scoped lang="scss">
 .gap-container {
   display: grid;
   gap: 1.5rem;
   grid-template-columns: repeat(auto-fit, minmax(min(15rem, 100%), 1fr));
   max-width: 100%;
+  visibility: hidden;
 }
 
 .centered {
   display: grid;
   place-content: center;
 }
+
+.visible {
+  visibility: visible;
+}
+
+// Stacks all children on top of each other
+.outer-container {
+  display: grid;
+  grid-template: 1fr / 1fr;
+  place-items: center;
+
+  > * {
+    grid-column: 1 / 1;
+    grid-row: 1 / 1;
+  }
+}
```

Comment:
what does this do? I'd think this would be a use case for absolute positioning?

Reply:
This stacks all the child divs on top of each other while still taking up space. So the size of the container ends up being the max size of the children.

Here's the stack overflow. I'll add it in a comment on top of this piece of code
https://stackoverflow.com/questions/1909648/stacking-divs-on-top-of-each-other

Reply:
Absolute elements don't take up space in a container though, so sizing that outer container with absolute elements is confusing?

Reply:
right right.  I wonder if this would be more clear if you used a named template then assigned every child's grid area to that named area?

Reply:
Style updated ! Thanks for the help on this Mary !